### PR TITLE
upgrading to visual-diff 11 and puppeteer 16

### PR DESCRIPTION
**Pull request by dbatiste**

### CODE REVIEW

File: visual-diff/README.md [hunk 1]
```diff
@@ -75,5 +75,5 @@ This action relies on the `@brightspace-ui/visual-diff`, `mocha` and `puppeteer`
 If you'd like to install the same versions the action is using locally, the specific versions currently used by the action are provided below:
 
 ```shell
-npm install @brightspace-ui/visual-diff@10 mocha@10 puppeteer@15  --no-save
+npm install @brightspace-ui/visual-diff@11 mocha@10 puppeteer@15  --no-save
```
Comment (dlockhart) [inline visual-diff/README.md:78]: This should be puppeteer 16, no?

Comment (eKoopmans) [inline visual-diff/README.md:78]: Do we want puppeteer 16 here too, or just within visual-diff?

Comment (dbatiste) [reply visual-diff/README.md]: Yep, ty.

Comment (dbatiste) [reply visual-diff/README.md]: Yep, ty.

**Review by svanherk — APPROVED** [review]

**Pull request merged**